backend/app/urls.py

```diff
@@ -0,0 +1,3 @@
+from django.urls import path
+
+urlpatterns = []
```

Comment:
always add extra line after  last line of code
```suggestion
urlpatterns = []

```